src/Header.css

```diff
@@ -0,0 +1,47 @@
+.Header {
```

Comment:
Can you follow this style guide? https://github.com/ubyssey/ubyssey.ca/wiki/SASS-Style-Guide

Something like:

```sass
.c-mobile-header {
  // ...
}
```